components/section-slideshow.jsx

```
@@ -0,0 +1,139 @@
+      {validImages.map((img, i) => {
+        // Xử lý URL hình ảnh
+        let imageUrl = '';
+        if (img.url) {
+          // Nếu đã có URL
+          imageUrl = img.url;
+        } else if (img.asset) {
+          // Nếu có asset trực tiếp, sử dụng URL tạm thời
+          imageUrl = `https://cdn.sanity.io/images/8ucvng19/production/${img.asset._ref.replace('image-', '').replace(/(-[a-z]+)$/, '.$1').replace('-', '.')}`;
+        } else if (img.image && img.image.asset) {
+          // Nếu có asset trong trường image, sử dụng URL tạm thời
+          imageUrl = `https://cdn.sanity.io/images/8ucvng19/production/${img.image.asset._ref.replace('image-', '').replace(/(-[a-z]+)$/, '.$1').replace('-', '.')}`;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)